theme/open/views/registration/forms/einwilligungserklaerung.hbs

```diff
@@ -15,13 +15,13 @@
 {{else}}
 	<label>
 		<input name="{{prefix}}privacyConsent" type="checkbox" value="true" required/>
 		<span>Ich erkläre mich damit einverstanden, dass meine personenbezogenen Daten entsprechend der <a href="{{@root.theme.documents.specificFiles.privacy}}" target="_blank" rel="noopener">Datenschutzerklärung</a> der Schule in der {{@root.theme.title}} sowie bei den von der Schule ausgewählten und weisungsgebundenen Lernmittelanbietern verarbeitet werden.</span>
 	</label>
-	<p class="explanation">Die Einwilligung und damit die Nutzung ist freiwillig, das heißt, wenn du deine Einwilligung nicht erteilst, führt diese für dich nicht zu Nachteilen im Unterricht. Eine Teilnahme am Unterricht, außerhalb der {{@root.theme.title}}, wird durch die Schule ermöglicht werden. Die Einwilligung kann jederzeit von dir widerrufen werden. Eine kurze E-Mail mit der Erklärung des Widerrufes, der Angabe des Vor- und Nachnamens, der Klasse und der Schule an folgende Adresse genügt: <a href="mailto:widerruf@schul-cloud.org?subject=Widerruf der Einwilligung">widerruf@schul-cloud.org</a>. Im Fall des Widerrufs löschen wir die auf Basis der Einwilligung gespeicherten personenbezogenen Daten unverzüglich, es sei denn, es besteht ein gesetzlicher Grund zur Aufbewahrung oder du willst vorher noch auf die gespeicherten Daten zugreifen. Weitere Hinweise zum Datenschutz kannst du auch unseren <a class="nav-link" target="_blank" href="https://docs.schul-cloud.org/x/GxfGBg">FAQ</a> entnehmen.</p>
+	<p class="explanation">Die Einwilligung und damit die Nutzung ist freiwillig, das heißt, wenn du deine Einwilligung nicht erteilst, führt diese für dich nicht zu Nachteilen im Unterricht. Eine Teilnahme am Unterricht, außerhalb der {{@root.theme.title}}, wird durch die Schule ermöglicht werden. Die Einwilligung kann jederzeit von dir widerrufen werden. Eine kurze E-Mail mit der Erklärung des Widerrufes, der Angabe des Vor- und Nachnamens, der Klasse und der Schule an folgende Adresse genügt: <a href="mailto:widerruf@schul-cloud.org?subject=Widerruf der Einwilligung" target="_blank">widerruf@schul-cloud.org</a>. Im Fall des Widerrufs löschen wir die auf Basis der Einwilligung gespeicherten personenbezogenen Daten unverzüglich, es sei denn, es besteht ein gesetzlicher Grund zur Aufbewahrung oder du willst vorher noch auf die gespeicherten Daten zugreifen. Weitere Hinweise zum Datenschutz kannst du auch unseren <a class="nav-link" target="_blank" href="https://docs.schul-cloud.org/x/GxfGBg">FAQ</a> entnehmen.</p>
```

Comment:
If it is a quick change maybe add a warning about the link opening in the new tab.

Reply:
I guess aria is not useful as the screen reader will only announce the "opens in a new tab" and via css would be quite some code I guess, but maybe we should decide how we want to proceed with links or documents opening in a new tab....🤔

Reply:
you can concatenate warnings and labels in an aria-label, then a SR should read both